jsonrpc/eth_endpoint.go

```diff
@@ -172,6 +207,37 @@ func (e *Eth) filterExtra(block *types.Block) error {
 	return nil
 }
 
+// CreateAccessList creates a EIP-2930 type AccessList for the given transaction.
+// Reexec and BlockNrOrHash can be specified to create the accessList on top of a certain state.
+func (e *Eth) CreateAccessList(arg *txnArgs, filter BlockNumberOrHash) (interface{}, error) {
+	header, err := GetHeaderFromBlockNumberOrHash(filter, e.store)
+	if err != nil {
+		return nil, err
+	}
+
+	transaction, err := DecodeTxn(arg, header.Number, e.store, true)
+	if err != nil {
+		return nil, err
+	}
+
+	res := &accessListResult{
+		Accesslist: transaction.AccessList(),
+		GasUsed:    argUint64(header.GasUsed),
+	}
+
+	return res, nil
+}
+
+// Returns the client coinbase address.
+func (e *Eth) Coinbase() (interface{}, error) {
+	h := e.store.Header()
+	if h == nil {
+		return nil, fmt.Errorf("header has a nil value")
+	}
+
+	return types.BytesToAddress(h.Miner), nil
+}
```

Comment:
I couldn't find the implementation for this function in the go ethereum and now thinking should we even expose it, if it is more related to the proof of work protocol. On the other hand, I don't mind having it like this...

Let's discuss with @goran-ethernal probably.

Reply:
I would remove this function.